graph_algorithm.py

```diff
@@ -373,24 +385,29 @@ def create_lcas_from_tuples(self, tuples):
             else:
                 nodes = self.clustering[cids[0]] | self.clustering[cids[1]]
             subG = self.G.subgraph(nodes)
-            from_score = ct.cid_list_score(subG, self.clustering,
-                                           self.node2cid, cids)
+            from_score = ct.cid_list_score(subG, self.clustering, self.node2cid, cids)
             a = lca.LCA(subG, self.clustering, cids, from_score)
             self.cid2lca.add(a)
             self.queues.add_to_S(a)
             num_created += 1
-        logger.info("Created %d new LCAs" % num_created)
+        logger.info('Created %d new LCAs' % num_created)
 
     def run_main_loop(self, iter_num=0, max_iterations=None):
         halt_requested = False
         should_pause = False
         converged = False
 
-        while (max_iterations is None or iter_num < max_iterations) and\
-              not (halt_requested or should_pause or converged):
+        # TODO: Added by JP due to linting errors
+        should_wait_for_edges = False
```

Comment:
These boolean flags (`should_wait_for_edges` and `stop_requested`) were referenced in the logic below but never defined.  I've added them at the top of `run_main_loop` but they are static and simply defined to be False